.buildkite/scripts/run_standard_models.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+## remove failure for testing
+set -euo pipefail
+
+apt-get update
+apt-get install libsasl2-dev
+
+python3 -m venv venv
+. venv/bin/activate
+pip install --upgrade pip setuptools
+pip install -r integration_tests/requirements.txt
+mkdir -p ~/.dbt
+cp integration_tests/ci/sample.profiles.yml ~/.dbt/profiles.yml
+
+db=$1
+echo `pwd`
+cd integration_tests
+dbt deps ## Install all packages needed
+
+shift ## Skips the first argument (warehouse) and moves to only looking at the data model arguments
+
+for model in "$@" ## Iterates over all non warehouse arguments
```

Comment:
I _think_ I used model because at one point I had two for loops and had package in the second one. Now that I condensed it into one loop I agree that using package would be much more consistent and less confusing.

Reply:
Updated